src/ServicePulse.Host/app/js/views/failed_messages/controller.js

```diff
@@ -45,6 +44,11 @@
             vm.sortButtonText = (sort === 'message_type' ? "Message Type" : "Time of Failure") + " " + (direction === 'asc' ? "ASC" : "DESC");
         }
 
+        var storeSortOption = function(sort, direction) {
```

Comment:
Maybe instead of using `store` a better word for these types of things would be `save`?